### ISSUE TITLE
IPv6 Loopback /64 route is not getting advertised to the neighbors

### ISSUE THREAD

**Issue by SuvarnaMeenakshi** [sonic-net/sonic-buildimage#15709]
#### Steps to reproduce the issue:
1. bring up a VS testbed with latest master branch image
./testbed-cli.sh -t vtestbed.yaml -m veos_vtb -k ceos add-topo vms-kvm-t0 password.txt

IPv6 loopback of SONiC VS is not reachable from ceos neighbors.

#### Describe the results you received:
IPv6 Loopback is not advertised to ceos neighbors.
Loopback0 configuration in config_db:
"Loopback0|FC0A:1::32/128": {}

vtysh output:
```
vlab-01# show bgp ipv6 fc0a:1::/64
BGP routing table entry for fc0a:1::/64, version 0
Paths: (1 available, no best path)
  Not advertised to any peer
  Local
    :: (inaccessible) from :: (10.1.0.32)
      Origin IGP, metric 0, weight 32768, invalid, sourced, local
      Last update: Mon Jul  3 23:00:48 2023

 ```

Error log in zebra.log
```
0:254:fc0a:1::/64: Failed to enqueue dataplane install
```
Static route configured for loopback /64 route is not installed correctly in frr, but is installed in kernel.
```
vtysh -c "show ipv6 route nexthop-group"

..q - queued..

 

C>* fc00::7c/126 (526) is directly connected, PortChannel104, 00:46:17
C>* fc02:1000::/64 (453) is directly connected, Vlan1000, 00:46:20
S>q fc0a:1::/64 [1/0] (648) is directly connected, Loopback0, weight 1, 00:05:19
C>* fc0a:1::32/128 (465) is directly connected, Loopback0, 00:46:20
```
```
ip -6 route show fc0a:1::/64
fc0a:1::/64 nhid 43 dev Loopback0 proto 196 metric 20 pref medium
```

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by SuvarnaMeenakshi** [comment]
@arlakshm @prsunny @StormLiangMS please check

**Comment by tjchadaga** [comment]
@SuvarnaMeenakshi - should be fixed by https://github.com/sonic-net/sonic-buildimage/pull/15688. Please verify

**Comment by SuvarnaMeenakshi** [comment]
Verified image with fix:
Issue is not seen.
```
vtysh -c "show ipv6 route nexthop-group" 
..
C>* fc00::7c/126 (349) is directly connected, PortChannel104, 00:30:00
**S>* fc00:1::/64 [1/0]** (224) is directly connected, Loopback0, weight 1, 00:30:05
C>* fc00:1::32/128 (223) is directly connected, Loopback0, 00:30:05
C>* fc02:1000::/64 (232) is directly connected, Vlan1000, 00:30:00
```
```
admin@vlab-01:~$ vtysh -c "show bgp ipv6 fc00:1::/64"
BGP routing table entry for fc00:1::/64, version 1
Paths: (1 available, best #1, table default)
  Advertised to non peer-group peers:
  fc00::72 fc00::76 fc00::7a fc00::7e
  Local
    :: from :: (10.1.0.32)
      Origin IGP, metric 0, weight 32768, valid, sourced, local, best (First path received)
      Last update: Wed Jul  5 20:57:42 2023
```
ping from neighbor ceos works - 
```
ARISTA01T1#ping fc00:1::32
PING fc00:1::32(fc00:1::32) 72 data bytes
80 bytes from fc00:1::32: icmp_seq=1 ttl=64 time=1.33 ms
80 bytes from fc00:1::32: icmp_seq=2 ttl=64 time=0.913 ms
80 bytes from fc00:1::32: icmp_seq=3 ttl=64 time=0.574 ms
80 bytes from fc00:1::32: icmp_seq=4 ttl=64 time=0.556 ms
80 bytes from fc00:1::32: icmp_seq=5 ttl=64 time=0.643 ms

--- fc00:1::32 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4ms
```
